rbac/providers/azure/initial_inbound_sync.py

```diff
@@ -177,14 +215,18 @@ def insert_user_to_db(users_dict):
 
 def initialize_aad_sync():
     """Initialize a sync with Azure Active Directory."""
+    LOGGER.info("connecting to RethinkDB...")
+    connect_to_db()
+    LOGGER.info("Successfully connected to RethinkDB!")
+
     LOGGER.info("Inserting AAD data...")
 
     LOGGER.info("Getting Users...")
     users = fetch_users()
     if users:
         insert_user_to_db(users)
-        while '@odata.nextLink' in users:
-            users = fetch_next_payload(users['@odata.nextLink'])
+        while "@odata.nextLink" in users:
```

Comment:
Please extract "@odata.nextLink" to a constant since it is referenced in multiple places

Reply:
@nadiabahrami @jbobo I am challenging both of you to produce documentation from the internet stating that extracting this value into a function-scoped variable (per discussion about avoiding global NS and fact that constants don't exist in Py) is a Python antipattern. I am finding the contrary in their best practices docs:
https://docs.python-guide.org/writing/style/#idioms
See: "Create an ignored variable" and observe the cases that follow where (even single use) strings are assigned to variables having meaningful names. 
Re: Constant: One nice idea I saw, in addition, was to put it in a function ( eg getNextPageKey()) if mutability is the concern.